rs/dre-canisters/trustworthy-node-metrics/src/trustworthy-node-metrics/src/rewards_manager.rs

```diff
@@ -0,0 +1,101 @@
+use chrono::{DateTime, Duration, Utc};
+
+use crate::types::{NodeMetrics, Rewards, TimestampNanos};
+
+#[derive(Debug)]
+pub struct DailyFailureRate {
```

Comment:
How about naming it to something more specific, e.g. `ProposeBlockFailureRateDaily`?